src/components/atoms/button/ButtonWhite.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export default function Button({ label, path }) {
+
+    return (
+        <>
+            <button className={`bg-white rounded-3xl w-32 font-medium mr-4 h-8 my-auto`}><Link href={path}>{label}</Link></button>
+        </>
+    )
+}
```

Comment:
Does your editor auto format code on save? If not I recommend figuring out how to enable it. It can improve readability and consistency.

Upon formatting on save, I get this:
```
export default function Button({ label, path }) {
  return (
    <>
      <button
        className={`my-auto mr-4 h-8 w-32 rounded-3xl bg-white font-medium`}
      >
        <Link href={path}>{label}</Link>
      </button>
    </>
  )
}
```

Reply:
when I select format I get this error: 
![Screenshot 2024-06-26 at 5 06 01 PM](https://github.com/virufy6/virufy6.github.io/assets/127047078/38169f7b-d24f-4493-97d0-21ed6c939445)

Reply:
Please install this: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
ok I got it working, thanks